source/docs/software/odometry.rst

```diff
@@ -66,18 +66,12 @@ The displacement (or change in position) of the left sensor
 is :math:`\Delta x_l` and the displacement of the right sensor
 is :math:`\Delta x_r`. The lateral distance between these two sensors
 is called the trackwidth. This is very important for determining angle
-for turning approximations. This value will need to be
-tuned, which means tested repeatedly and then brought to some
-converging value that is close to the actual measurement.
+for turning approximations. This value :math:`L` (for lateral)
+will need to be tuned, which means tested repeatedly and
+then brought to some converging value that is close to the
+actual measurement.
 
-.. figure:: images/odometry/andrew-bot-forward-offset-quarter.b5714ec5.jpg
-    :alt: The lateral distance, forward offset, and location of the sensors
-
-    `17508 Rising Tau's 2019/20 Skystone Bot <https://www.learnroadrunner.com/dead-wheels.html#three-wheel-odometry>`_
```

Comment:
Something is up with this image. Could be the .b5714ec5 in the name or could just be too large. Seems fixable though.

Reply:
So it isn't the name.

Reply:
Resolved by converting image to png from jpg.